packages/localizations/src/ms-MY.ts

```
@@ -14,155 +14,176 @@ import type { LocalizationResource } from '@clerk/types';
+      formButtonSubmit: 'Cipta organisasi baharu',
+      formButtonReset: 'Batal',
+    },
+    chooseOrganization: {
+      title: 'Pilih organisasi',
+      subtitle: 'Sertai organisasi sedia ada atau cipta yang baharu',
+      suggestionsAcceptedLabel: 'Menunggu kelulusan',
+      action__createOrganization: 'Cipta organisasi baharu',
+      action__suggestionsAccept: 'Minta untuk menyertai',
+      action__invitationAccept: 'Sertai',
+    },
+  },
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)